### ISSUE TITLE
[BUG] New warning generated with 02/25/2021 Bugfix.

### ISSUE THREAD

**Issue by bill-orange** [MarlinFirmware/Marlin#21203]
Bug Description: compilation is successful with BugFix but with this warning at the beginning of the compile. 

`Can not remove temporary directory `C:\Users\William\Documents\Marlin\Marlin\.pio\build`. Please remove it manually to avoid build issues`

Manually deleting the build folder makes no difference.  Compiling for a different microcontroller makes no difference.  Compilation with 2.0.x does not generate this error.  I did not see this error until last night.  It did not appear to be an issue with prior Bugfix builds. 

Configuration Files ( I also included my platformio.ini):

[Marlin.zip](https://github.com/MarlinFirmware/Marlin/files/6050969/Marlin.zip)

Steps to Reproduce:

1. Compile
2. Observe warning
3. Manually delete build and recompile
4. Observe warning

Expected behavior:  No warning indicating a potential defective build.

Actual behavior: Warning as noted

#### Additional Information

One user reported in the Facebook group that this did not occur when using AutoBuild.  I am not an AutoBuild user.

**Comment by Foxies-CSTL** [comment]
I have the same issue with last bump (2021-02-26)
![image](https://user-images.githubusercontent.com/24864147/109335548-7af5f680-7862-11eb-9882-3ed916ceb71d.png)

**Comment by BirkBinnard** [comment]
I got that error too - fixed it by setting VSCode to operate in Administrator mode. But then I got the Platformio rebuilding cache message that never goes away. Fixed that by deleting everything from the platformio\.cache directory, deleting and reinstalling PlatformIO. That let Marlin my Marlin config compile. 

But when I tried to boot it on my printer I got the dreaded "All Low" error message (I was never able to find out what it means.) So I reverted to an earlier BugFIx version that does compile and boot OK. 

Unfortunately it doesn't work properly, but at least it boots without errors.

**Comment by rhapsodyv** [comment]
I find it... it's a marlin config issue that is generating a exception on some internal PIO method...

Here:
```
Invalid '/Users/victor/Development/Marlin/platformio.ini' (project configuration file): 'No section: 'LERDGEX''
```

I'm sending a PR to fix that

**Comment by rhapsodyv** [comment]
Fixed: #21205

**Comment by ellensp** [comment]
closing as is merged
